### PR TITLE
[MM-17066] use ALT+click to mark a post as unread

### PR DESCRIPTION
**Pull request by Willyfrog**
#### Summary

Use ALT+click on a post to mark it as unread

#### Ticket Link
[MM-17066](https://mattermost.atlassian.net/browse/MM-17066)

#### Related Pull Requests
Requires https://github.com/mattermost/mattermost-webapp/pull/3427

### CODE REVIEW

**Comment by hmhealey** [comment]
Changes specific to this PR: https://github.com/Willyfrog/mattermost-webapp/compare/MM-17065...Willyfrog:MM-17066

File: components/post_view/post/post.jsx [hunk 1]
```diff
@@ -111,10 +113,14 @@ export default class Post extends React.PureComponent {
 
     componentDidMount() {
         this.postRef.current.addEventListener(A11yCustomEventTypes.ACTIVATE, this.handleA11yActivateEvent);
+        document.addEventListener('keydown', this.handleAlt);
```
Comment (enahum) [inline components/post_view/post/post.jsx]: @Willyfrog I think this needs to be handled in the A11 controller, @devinbinnie can you confirm?

Comment (devinbinnie) [reply components/post_view/post/post.jsx]: I don't think it needs to be, both listeners are at the `document` level so they should both fire. I don't think anything should interfere with how the handler here works, but @deanwhillier might have more information.

Comment (Willyfrog) [reply components/post_view/post/post.jsx]: @enahum @deanwhillier do you feel we need to change it here?

Comment (deanwhillier) [reply components/post_view/post/post.jsx]: A11Y controller uses the capture phase of events so regular bubble event listeners like these don't prevent it from receiving user events.

Comment (deanwhillier) [reply components/post_view/post/post.jsx]: Even if `event.preventDefault()` is used.

**Pull request merged**